[core]/es_extended/client/functions.lua

```diff
@@ -897,6 +898,9 @@ function ESX.Game.SetVehicleProperties(vehicle, props)
     end
     if props.modFrontWheels ~= nil then
         SetVehicleMod(vehicle, 23, props.modFrontWheels, false)
+    end
+	if props.modCustomWheels ~= nil then
+         SetVehicleMod(vehicle, 23, props.modFrontWheels, true) 
```

Comment:
Hi! This is bad,  you should use "props.modCustomWheels". I think is typo, please fix this typo.